e3nn/tensor_product.py

```diff
@@ -109,3 +109,21 @@ def forward(self, input):
         output = input
         output = self.tp(output, output)
         return self.dot(output, input)
+
+
+class LearnableMultiplicityBispectrum(torch.nn.Module):
+    def __init__(self, mul_in, single_mul_Rs_in, mul_hidden, mul_out):
```

Comment:
Why not ask for `Rs_hidden` in argument? Do we really need to impose the kind of representation?

Reply:
That's true. The same could be said for `LearnableBispectrum`. I'll modify for both.